src/App.tsx

```diff
@@ -15,48 +17,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  Letters = 'letters',
+  Length = 'length',
+  Initial = '',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: SortType,
+  reverseField = false,
```

Comment:
You must accept this data through destructuring, and then type the variables through interface or type, because if you rearrange sortField and isReversed when calling the function, your code will break
If you call the function like this
  const visibleGoods = getPreparedGoods(
     goodsFromServer, reverseField, sortField
   );
There will be an error